doc/nrf/protocols/thread/device_types.rst

```diff
@@ -0,0 +1,133 @@
+.. _thread_device_types:
+
+Thread device types
+###################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Thread devices can be either Full Thread Devices (FTD) or Minimal Thread Devices (MTD).
+The devices are divided based on their ability to exist in a network without a parent, determined by whether the device maintains a routing table (doesn't need a parent) or not (needs a parent).
+FTDs are the devices that maintain a routing table, while MTDs do not, forwarding all their messages to a parent instead.
+
+Another type of Thread device classification is based on the device's ability to route messages:
+
+* Routers can route messages, and they can function as parents to other Thread devices.
+* End Devices cannot route messages, and they cannot function as parents.
+
+Based on this classification, FTDs can be either Routers or End Devices, while MTDs can only be End Devices.
+
+For additional information about Thread device types, see the documentation for `Device Types on OpenThread portal`_.
+
+.. _thread_types_ftd:
+
+Full Thread Devices
+*******************
+
+In order to maintain the routing table, FTDs keep their radio on at all times.
```

Comment:
Main reason why FTDs (but in fact also MTDs (like MED or SED during attachment, scanning)) - is that they need to be ready to receive messages asynchronously (from other devices). Maintaining routing table is a separate topic.